akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/VoiceApp.scala

```diff
@@ -10,11 +10,24 @@ import org.bigbluebutton.core2.message.senders.MsgBuilder
 
 object VoiceApp {
 
-  def genRecordPath(recordDir: String, meetingId: String, timestamp: Long): String = {
+  def genRecordPath(
+      recordDir:       String,
+      meetingId:       String,
+      timestamp:       Long,
+      recordAudioCode: String
+  ): String = {
+    val validCodecs = Set("wav", "ogg")

```

Comment:
We're planning to enable `mod_opusfile` in our freeswitch build, so please also allow "opus" as a valid format.